src/test/java/utils/seleniumWrapper.java

```diff
@@ -0,0 +1,35 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public class seleniumWrapper {
+
+    WebDriver driver = new EdgeDriver();
```

Comment:
could you please make it a public variable like this:
public WebDriver driver;

We can then create a function in this class where we initialise it with the required browser's driver.

Reply:
changed to public variable